src/Network/HttpClient.php

```diff
@@ -102,26 +83,14 @@ private function getClient()
      * Sends request to Google Analytics.
      *
      * @param string $url
-     * @param SingleParameter[] $singleParameters
-     * @param CompoundParameterCollection[] $compoundParameters
      * @param boolean $nonBlocking
      * @return AnalyticsResponse
      */
-    public function post($url, array $singleParameters, array $compoundParameters, $nonBlocking = false)
+    public function post($url, $nonBlocking = false)
```

Comment:
@jorgeborges IMO this is a BC-Break and should not be released in a feature version (`2.3.0`)